src/GameLogic/ItemPriceCalculator.cs

```diff
@@ -438,38 +573,6 @@ private static long CalculateBuyingPrice(Item item, byte durability)
             price = MaximumPrice;
```

Comment:
Another price bug seems to be the guardian (380 item) option. The client doesn't value the option. But the server sources I've checked do. What to do?
![Screenshot from 2024-12-04 15-26-58](https://github.com/user-attachments/assets/dc8e5467-ebb6-47e4-b3e8-fa17bb7fc92f) ![Screenshot from 2024-12-04 15-26-44](https://github.com/user-attachments/assets/14d6bd7b-c07e-4a0a-8799-573abfc9b729)